docs/criteria/accelerators/beta.md

```diff
@@ -87,23 +85,41 @@ Indexes are not required for test coverage, but can be introduced if required fo
   - [ ] Struct
   - [ ] Decimal128/Decimal256
 - [ ] Integration tests to cover "On Conflict" behaviors.
-- [ ] A reproducible test script that can load a variety of datasets into accelerators in both file/embedded memory mode.
-  - Loads and runs these benchmarks:
-    - [ ] TPCH-SF10 (embedded memory/file)
-    - [ ] TPCH-SF100 (file)
-    - [ ] TPCDS-SF10 (embedded memory/file)
-    - [ ] TPCDS-SF100 (file)
-    - [ ] ClickBench (memory/file)
-
-### Docs
-
-- [ ] Doc/Specification include all required information for a user to set up the accelerator.
-- [ ] Include all known issues/limitations.
-- [ ] Include any exceptions made to allow this accelerator to reach Beta quality (e.g. if a particular data type cannot be supported by the accelerator).
-- [ ] Easy to follow quickstart.
-- [ ] All remaining [minor](#bug-levels) TPCDS, TPCH and ClickBench bugs are raised as issues.
-
-### Data correctness
-
-- [ ] TPCH SF10 loaded into memory, returned results are identical across source/federated/accelerated queries for all 21 TPCH queries and TPC simple queries.
-- [ ] TPCH SF100 loaded into file, returned results are identical across source/federated/accelerated queries for all 21 TPCH queries and TPC simple queries.
+
+#### TPC-H
+
+- [ ] End-to-end test to cover accelerating TPC-H SF1 data from S3 and benchmarking TPC-H queries (official and simple).
+  - [ ] All supported modes should run all queries with no [Major Bugs](#definitions).
+- [ ] A test script exists that can load TPC-H SF10 and TPC-H SF100 data into this accelerator in all supported modes.
+- [ ] The accelerator can load TPC-H SF10 in all supported modes, and can run all queries with no [Major Bugs](#definitions).
+- [ ] The accelerator can load TPC-H SF100 in either file or direct database mode, and can run all queries with no [Major Bugs](#definitions).
+
+#### TPC-DS
+
+- [ ] End-to-end test to cover accelerating TPC-DS SF1 data from S3 and benchmarking TPC-DS queries (official and simple).
+  - [ ] All supported modes should run all queries with no [Major Bugs](#definitions).
+- [ ] A test script exists that can load TPC-DS SF10 and TPC-DS SF100 data into this accelerator in all supported modes.
+- [ ] The accelerator can load TPC-DS SF10 in all supported modes, and can run all queries with no [Major Bugs](#definitions).
+- [ ] The accelerator can load TPC-DS SF100 in either file or direct database mode, and can run all queries with no [Major Bugs](#definitions).
+
+#### ClickBench
+
+- [ ] A test script exists that can load ClickBench data into this accelerator in either file or direct database mode.
+- [ ] The accelerator can load ClickBench in either file or direct database mode, and all queries are attempted.
+  - [ ] All query failures should be logged as issues. No bug fixes are required for ClickBench
+
+#### Data correctness
+
```

Comment:
Should we just use all supported mode here? Since memory / file mode criteria doesn't apply to postgres

Reply:
I've changed the wording for "file" here into "file or direct database mode". We don't want "all supported modes", because memory mode cannot accelerate SF100 unless we run it on systems with extreme amounts of memory.